### PR TITLE
2.0.2

### PR DESCRIPTION
**Pull request by frankh**
Edit previous changes to make it properly backwards compatible

### CODE REVIEW

**Comment by JeanFred** [comment]
Can you expand the commit message please ? :)

**Comment by frankh** [comment]
huh...that was supposed to be 2 separate commits

File: cabot_alert_hipchat/models.py [hunk 1]
```diff
@@ -84,20 +84,27 @@ def _send_hipchat_alert(self, message, color='green', sender='Cabot'):
             if env.get('HIPCHAT_DOMAIN'):
                 logger.warn('Both HIPCHAT_URL and HIPCHAT_DOMAIN are present. Ignoring HIPCHAT_URL.')
```
Comment (JeanFred) [inline cabot_alert_hipchat/models.py]: And then nothing happens?

Comment (frankh) [reply cabot_alert_hipchat/models.py]: yep, just noticed this...

**Comment by JeanFred** [comment]
LGTM code wise. Do we need an intermediate version 2.0.1 then? And can we squash the typo fix ? :)

**Comment by frankh** [comment]
@JeanFred annoyingly i already pushed 2.0.1 to pypi, so we have to keep it

**Comment by frankh** [comment]
squashed

**Comment by JeanFred** [comment]
LGTM

**Pull request merged**